### ISSUE TITLE
Cannot reproduce the metric in paper

### ISSUE THREAD

**Issue by Guocode** [TAMU-VITA/GLNet#3]
I have followed the settings in paper and run the code, but the mIoU of  "global only" is only 63.49 but your paper shows 66.4, is there any tricks or can you tell the way to get the metric to 66.4.

I train the train_deep_globe.py with the following settings.
```
--n_class
7
--data_path
"E:/land-train/"
--model_path
"saved_models/"
--log_path
"runs/"
--task_name
"fpn_global.508_4.28.2019_lr2e5"
--mode
1
--batch_size
6
--sub_batch_size
6
--size_g
508
--size_p
508
```

**Comment by chenwydj** [comment]
Hi! As mentioned in our paper we used learning rate of 1e-4 for the global branch training. You may have a try on this. Other details like to use focal loss and do not consider the class #0 in the metric.

**Comment by lamhoangtung** [comment]
Hi @Guocode, how did you get the repo up and running :P ?

**Comment by 864546664** [comment]
hi ,didi you run the code on deepglobe?how do you solve the bug in one_hot?thanks

**Comment by chenwydj** [comment]
Hi! We updated the instructions for training and evaluation in the Readme, and also uploaded the pretrained model for the Deep Globe. Please take a look and give a try! :)
